Cargo.toml

```
@@ -14,7 +14,7 @@ license = "MIT OR Apache-2.0"
 # anything to the list of stable crates.
 # Only bump  0.x.* to 0.(x+1).0 on any nearcore release as nearcore does not guarantee
 # semver compatibility. i.e. api can change without a protocol upgrade.
-version = "0.20.1"
+version = "0.30.0"
```

Set RC version to crates and publish them manually as, I believe that the current flow only pushes on mainnet release.
```suggestion
version = "0.30.0-rc.1"
```